### DISCUSSION TITLE
Error trying to start Ollama via Powershell

### DISCUSSION THREAD

**Discussion by OkayHero** [nicobrauchtgit/obsidian-Smart2Brain#126] [Q&A]
Getting this error
Error: listen tcp 127.0.0.1:11434: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted.
when trying to run this command
$env:OLLAMA_ORIGINS="app://obsidian.md*"; ollama serve
checking into the error, it looks like obsidian and ollama are trying to use those ports already, I think I might just delete everything and start again, but just in case someone else runs into this, I'm posting here in case someone else knows how to fix it.

EDIT: Found on reddit, just closed Ollama, and then ran the command, and everything worked.

**Comment by Leo310** [comment]
This error occurs when you already have an Ollama instance running on your computer. 
To resolve this, you have to quit Ollama over your task manager or the system tray.
Then retry running the command.
